packages/material/src/mui-controls/MuiSelect.tsx

```diff
@@ -55,9 +72,9 @@ export const MuiSelect = React.memo((props: EnumCellProps & WithClassname) => {
       fullWidth={true}
     >
       {[<MenuItem value='' key={'empty'} />].concat(
-        options.map(optionValue => (
-          <MenuItem value={optionValue} key={optionValue}>
-            {optionValue}
+        keyValueLabels.map(option => (
+          <MenuItem value={option.value} key={option.key}>
```

Comment:
Keys are uses to [uniquely identify](https://reactjs.org/docs/lists-and-keys.html#keys) elements in React. Therefore there is no need to customize it. Just using the value is good enough, as the value [should be unique](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.1.2).

Reply:
Just using value without the label fails at ajv validation as it expects the value to be one of the allowed enums.

Reply:
I'm not sure we're talking about the same thing. What I meant is that the separate `option.key` is not needed, e.g.
```suggestion
          <MenuItem value={option.value} key={option.value}>
```
would also be fine.